From 3964634a6bf3927e714fa72d6435c6ced3352589 Mon Sep 17 00:00:00 2001
From: Shawn Lin <shawn.lin@rock-chips.com>
Date: Thu, 30 Mar 2017 09:30:50 +0800
Subject: [PATCH] mmc: core: use default generic cmd6 timeout for flushing
 cache

We need a luxury timeout once needing some extra time to
wait for flushing cache.

Change-Id: I8cd4015f30fa45cacdb984f0461b1ad8ee6cba7d
Signed-off-by: Shawn Lin <shawn.lin@rock-chips.com>
---
 drivers/mmc/core/core.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mmc/core/core.c b/drivers/mmc/core/core.c
index 31fc5362f99a..a68f956548e4 100644
--- a/drivers/mmc/core/core.c
+++ b/drivers/mmc/core/core.c
@@ -2806,7 +2806,7 @@ int mmc_flush_cache(struct mmc_card *card)
 			(card->ext_csd.cache_size > 0) &&
 			(card->ext_csd.cache_ctrl & 1)) {
 		err = __mmc_switch(card, EXT_CSD_CMD_SET_NORMAL,
-				   EXT_CSD_FLUSH_CACHE, 1, 1,
+				   EXT_CSD_FLUSH_CACHE, 1, 0,
 				   true, false, false);
 		if (err)
 			pr_err("%s: cache flush error %d\n",
-- 
2.35.3

